src/components/Video/Video.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect, useRef } from 'react';
+import { View } from '@aws-amplify/ui-react';
+
+interface VideoProps {
+  autoPlay?: boolean;
+  muted?: boolean;
+  loop?: true;
+  src: string;
+  testId?: string;
+}
+
+export const reducedMotionMediaQuery =
+  '(prefers-reduced-motion: no-preference)';
+
+/**
+ * @description The Video component defaults to a muted, auto play video.
+ * Currently, we also assume the surrounding content will adequately describe
+ * the video so we default to aria-hidden="true".
+ */
+export const Video = ({
+  autoPlay = true,
+  muted = true,
+  loop = true,
+  src,
+  testId,
+  ...rest
+}: VideoProps) => {
+  /**
+   * Assume user prefers reduced motion until we can check
+   * in the useEffect for the media query match, otherwise Next SSG
+   * will consider 'window' undefined.
+   */
+  const [prefersReducedMotion, setPrefersReducedMotion] =
+    useState<boolean>(true);
+
+  const videoRef = useRef<HTMLVideoElement>(null);
+
+  useEffect(() => {
+    const mediaQueryList = window.matchMedia(reducedMotionMediaQuery);
+
+    setPrefersReducedMotion(
+      !window.matchMedia(reducedMotionMediaQuery).matches
```

Comment:
nit: Could we reuse mediaQueryList from above for this line?
```suggestion
      !mediaQueryList.matches
```

Reply:
Thanks for the catch, fixed in latest